index.html

```diff
@@ -226,7 +226,47 @@ <h3 class="list_text">Skills</h3>
     </section>
     <!-- about section ends -->
 
+    <!-- Contact Form section starts here -->
+    <section class="form-container">
+        <div class="img_geometry">
+            <img src="img/image_geometry_1.png" alt="" class="shape1">
+        </div>
+        <header class="title_font">Get Started</header>
+        <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Exercitationem, odio? Lorem ipsum dolor sit amet.</p>
+        <form class="form" action="https://formspree.io/f/xqkjgdvv" method="POST">
+            <ul>
+                <li>
+                    <input type="text" name="name" placeholder="Your Name" maxlength="30" required>
+                </li>
+                <li>
+                    <input type="email" name="email" placeholder="Your Email" required>
+                </li>
+                <li>
+                    <textarea name="message" placeholder="Enter your message here..." cols="25" rows="5" maxlength="500"></textarea>
+                </li>
+                <li>
+                    <button type="submit">Start collaboration</button>
```

Comment:
- [ ] as part of the requirements, kindly add the button interactions as mentioned in Figma 🙏 

![image](https://user-images.githubusercontent.com/89282221/199473631-e7a7ebb2-fa72-4013-ac1e-79a30670bd9b.png)